cli/operator/node.go

```diff
@@ -112,7 +114,8 @@ var StartNodeCmd = &cobra.Command{
 		cfg.P2pNetworkConfig.Permissioned = permissioned
 		cfg.P2pNetworkConfig.WhitelistedOperatorKeys = append(cfg.P2pNetworkConfig.WhitelistedOperatorKeys, networkConfig.WhitelistedOperatorKeys...)
 
-		p2pNetwork := setupP2P(forkVersion, operatorData, db, logger, networkConfig)
+		messageValidator := validation.NewMessageValidator(networkConfig.Beacon)
```

Comment:
@nkryuchkov maybe better this to be setup inside the setupP2p and not here?

Reply:
@lior-blox that may be temporary, `NewMessageValidator` will receive more args and it would be easier to do it here. I'll leave a TODO to consider moving it to `setupP2P` depending on amount of args